### PR TITLE
Integration test connector

### PR DESCRIPTION
**Pull request by Logicon211**

### CODE REVIEW

**Review by yurii-klymenko — APPROVED** [review]

File: stix_shifter_modules/error_test/entry_point.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+class EntryPoint(BaseEntryPoint):
+
+    def __init__(self, connection={}, configuration={}, options={}):
+        super().__init__(connection, configuration, options)
+        self.set_async(False)
+        if connection:
+            self.setup_transmission_basic(connection, configuration)
+
+        self.add_dialect('default', default=True)
```
Comment (mdazam1942) [inline stix_shifter_modules/error_test/entry_point.py:12]: you can just set this instead `self.setup_translation_simple(dialect_default='default')` to be consistent with other connectors.

File: stix_shifter_modules/error_test/stix_translation/query_translator.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+START_STOP_PATTERN = "\s?START\s?t'\d{4}(-\d{2}){2}T\d{2}(:\d{2}){2}(\.\d+)?Z'\sSTOP\s?t'\d{4}(-\d{2}){2}T(\d{2}:){2}\d{2}.\d{1,3}Z'\s?"
+
+ERROR_TYPE_PARSE_EXCEPTION = 'parse_exception'
+ERROR_TYPE_TRANSFORM_EXCEPTION = 'transform_exception'
+
+class QueryTranslator(EmptyQueryTranslator):
+
+    def parse_query(self, data):
+        print('**IN PARSE QUERY')
+        print(self.options.get('error_type'))
```
Comment (mdazam1942) [inline stix_shifter_modules/error_test/stix_translation/query_translator.py:13]: you can replace print statements with our logger utility `from stix_shifter_utils.utils import logger` unless you have specific case for print statements.

Comment (Logicon211) [reply stix_shifter_modules/error_test/stix_translation/query_translator.py:13]: I missed this one. Will remove

File: stix_shifter_modules/error_test/entry_point.py (hunk 1 above)
Comment (Logicon211) [reply stix_shifter_modules/error_test/entry_point.py:12]: Am I replacing the entire `self.setup_transmission_basic(connection, configuration)` here? Won't I still need to pass in connection and configuration?

Comment (mdazam1942) [reply stix_shifter_modules/error_test/entry_point.py:12]: sorry should have mentioned, this is just to replace `self.add_dialect('default', default=True)`

Comment (mdazam1942) [reply stix_shifter_modules/error_test/entry_point.py:12]: or you can just leave it. which should also works.

**Review by mdazam1942 — APPROVED** [review]

**Pull request merged**